pymc_marketing/mmm/transformers.py

```diff
@@ -826,10 +826,10 @@ def tanh_saturation_baselined(
     return gain * x0 * pt.tanh(x * pt.arctanh(r) / x0) / r
 
 
-def michaelis_menten(
+def michaelis_menten_function(
```

Comment:
This is only used in the functions we will deprecate, right?

Reply:
> This is only used in the functions we will deprecate, right?

Yes, they are only used in the methods linked in #1055. 
If we remove those, there is no need to define the extra function.

Reply:
Mind closing that now?

Reply:
Hi @wd60622,

having doubts about completely removing functions in #1055. 

As an example, take `estimate_menten_parameters`. It was introduced in #329, and never used outside of the current test. I am guessing the original purpose was to double check that the introduced function gives correct results. If we remove the tests, we would never be checking that again.

As alternative I've simplified the PR to wrap only `MichaelisMentenSaturation.function` with `pt.as_tensor_variable`.

Note that the newly introduced fixture now passes the tests

Reply:
If we need to remove tests, that is fine. 

This is only saturation transformation which needs to be defined with two functions (which I find silly). Just cosolidate